src/particles/wakefields/ChargeBinning.H

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2022-2023 The Regents of the University of California, through Lawrence
+ *           Berkeley National Laboratory (subject to receipt of any required
+ *           approvals from the U.S. Dept. of Energy). All rights reserved.
+ *
+ * This file is part of ImpactX.
+ *
+ * Authors: Alex Bojanich, Chad Mitchell, Axel Huebl
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef CHARGE_BINNING_H
+#define CHARGE_BINNING_H
+
+#include "../ImpactXParticleContainer.H"
+
+using namespace amrex;
```

Comment:
We cannot use namespace in header files, since this would propagate to all files that include the header and then cause collisions.
```suggestion
```

Please prefix AMReX types in this file explicitly with `amrex::`, e.g., `amrex::Real`.